test/parallel/test-vm-is-context.js

```diff
@@ -26,14 +26,16 @@ const vm = require('vm');
 
 assert.throws(function() {
   vm.isContext('string is not supported');
-}, TypeError);
+}, /TypeError/);
 
 assert.strictEqual(vm.isContext({}), false);
 assert.strictEqual(vm.isContext([]), false);
 
 assert.strictEqual(vm.isContext(vm.createContext()), true);
 assert.strictEqual(vm.isContext(vm.createContext([])), true);
 
-const sandbox = { foo: 'bar' };
-vm.createContext(sandbox);
-assert.strictEqual(vm.isContext(sandbox), true);
+{
```

Comment:
It might be useful if you want to reuse the `sandbox` variable in another scope but currently there is no need for it.